docs/reference/elasticsearch/rest-apis/api-conventions.md

```diff
@@ -1,8 +1,10 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster.html#cluster-nodes
```

Comment:
I think we need to leave the anchor out (the entire page contents are covered anyway)

```suggestion
  - https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster.html
```